### ISSUE TITLE
Control project needs an x64 build config on Windows.

### ISSUE THREAD

**Issue by fierval** [udacity/fcnd-issue-reports#293]
Simulator.vcxproj needs to have a functional x64 configuration, otherwise Windows 10 is not really supported. Solution here: https://github.com/fierval/FCND-Controls-CPP

**Comment by adrnp** [comment]
Were you able to run the Win32 (x86) build configuration?  If not, what was the error you were getting with that?

**Comment by fierval** [comment]
It built fine, but I could not run it due to an OpenGL error. The error may have gone away if I just rebooted - I saw it happen during x64 builds as well, and rebooting would fix it, so I'd say this is not a p1 bug, however, it's simple enough to provide an x64 config (see above) and since an overwhelming majority is running x64 systems these days, this should probably be the default. This is one of those annoying little things that make devs look silly in the eyes of a user, and since we are paying users it's probably worth fixing.
